src/bentoml/_internal/client/grpc.py

```diff
@@ -221,13 +222,16 @@ def _rpc_metadata(self) -> dict[str, dict[str, t.Any]]:
             )
         }
 
-    async def health(self, service_name: str, *, timeout: int = 30) -> t.Any:
+    async def async_health(self, service_name: str, *, timeout: int = 30) -> t.Any:
         return await self._invoke(
             method_name="/grpc.health.v1.Health/Check",
             service=service_name,
             _grpc_channel_timeout=timeout,
         )
 
+    def health(self, service_name: str, *, timeout: int = 30) -> t.Any:
+        return asyncio.run(self.async_health(service_name, timeout=timeout))
```

Comment:
Should we do something like what we do with runners and `anyio` to avoid issues with running loops?

Reply:
we will have to spawn an anyio worker right?